### PR TITLE
use rem instead of px

### PR DESCRIPTION
**Pull request by kmer2016**

### CODE REVIEW

**Review by achrafmohye — APPROVED** [review]
LGTM

File: ui-core/src/styles/inputs/passwordInput.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .password-input {
     .input {
-        width: 235px !important;
+        width: 14.6875rem !important; /* Converted from 235px */
```
Comment (Math-R) [inline ui-core/src/styles/inputs/passwordInput.css:3]: 3 decimals accuracy is enough

https://stackoverflow.com/questions/35492847/how-granular-are-css-rem-units

File: ui-core/src/styles/inputs/tokenInput.css [hunk 2]
```diff
@@ -1,49 +1,48 @@
...
-        min-height: 40px;
-        padding: 4px;
-        width: 344px;
-        border-radius: 4px;
-        box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.5) inset, 0px 0px 0px 1px rgba(148, 145, 142, 1) inset;
+        min-height: 2.5rem;
+        padding: 0.25rem;
+        width: 21.5rem;
+        border-radius: 0.25rem;
+        box-shadow: 0 0.0625rem 0.1875rem rgba(0, 0, 0, 0.5) inset, 0 0 0 0.0625rem rgba(148, 145, 142, 1) inset;
```
Comment (Math-R) [inline ui-core/src/styles/inputs/tokenInput.css:17]: same here

File: ui-core/src/styles/inputs/tokenInput.css [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
             .token-close-btn {
-                margin: 4px 4px 4px 0;
+                margin: 0.25rem 0.25rem 0.25rem 0;
                 @apply text-black-25;
             }
 
             &:hover {
                 cursor: pointer;
-                box-shadow: 0px 0px 0px 2px rgba(255, 255, 255, 0.5) inset, 0px 0px 0px 1px rgb(152, 192, 245) inset;
+                box-shadow: 0 0 0 0.125rem rgba(255, 255, 255, 0.5) inset, 0 0 0 0.0625rem rgb(152, 192, 245) inset;
```
Comment (Math-R) [inline ui-core/src/styles/inputs/tokenInput.css:45]: same here

File: ui-core/src/styles/inputs/tokenInput.css [hunk 4]
```diff
@@ -52,7 +51,7 @@
             }
 
             &:active {
-                box-shadow: 0px 0px 0px 1px rgb(32, 30, 222) inset;
+                box-shadow: 0 0 0 0.0625rem rgb(32, 30, 222) inset;
```
Comment (Math-R) [inline ui-core/src/styles/inputs/tokenInput.css:54]: same

**Review by Math-R — APPROVED** [review]
lgtm =)

**Pull request closed without merging**